docs/modules/trino/pages/usage-guide/connect_to_trino.adoc

```diff
@@ -78,3 +88,28 @@ connection = get_connection()
 
 run_query(connection, "show catalogs")
 ----
+
+In case you are using OpenID connect use the following connection definition:
+
+[source,python]
+----
+    connection = trino.dbapi.connect(
+        host="87.106.119.4", # You need to use an an externally reachable address, so that your browser can access it!
+        port=8443,
+        http_scheme="https",
+        verify=False, # For best security you can also provide a path to the trino root ca: "/stackable/secrets/trino-ca-cert/ca.crt",
```

Comment:
We should probably give the secure example, and then offer False in the comment.

Reply:
First off: I think the important part is documenting the secure path.

I personally would prefer to ship an working example, especially as sadly my feeling is many of the users are not knowledge enough to get the ca.crt, we have tracking issues to improve that situation, e.g. https://github.com/stackabletech/secret-operator/issues/410

Reply:
I get that the examples should be working, but can we not have both secure and working? (With autoTls)

Feel free to say no, and mark as resolved, I just wanted to discuss here so I can understand why not.

Reply:
>  but can we not have both secure and working? (With autoTls)

I execute the script on my local machine (or JupyterHub). So I first need to follow the workaround from https://github.com/stackabletech/secret-operator/issues/410, extract the ca.crt and place it on my local machine.
I can do that easily but I fear many users will have problems with that/are not able to do that.

So I don's see any config that work on all the developer machines out there :see_no_evil: 
But I can tell you that the python client is *very* good in annoying you with many many warnings :D

Reply:
Oh I see. It's just for local development from a python script. I thought it was configuring a deployed thing which should definitely use the secure example even if it is "harder".

So I'm fine with this.

Reply:
Oh yeah, this is *not* a k8s resource where we could point to a SecretClass, but a random Python script instead :+1: